sprints/sprint_1_ru.html

```diff
@@ -65,38 +65,6 @@ <h2>Результаты</h2>
       Листинг правил доступен в описанни <a href="https://github.com/Neo23x0/sigma/pull/554">Pull Request</a> в репозитории Sigma.<br>
       Итоги опубликованы в статьях на <a href="https://habr.com/ru/post/482642/">Хабре</a> (RU) и <a href="https://medium.com/oscd/oscd-threat-detection-sprint-1-c42317e06771">Medium</a> (EN).
       <br><br>
```

Comment:
please update the russian page accordingly 🙏

Reply:
Done, thx